base/_mixins.scss

```diff
@@ -57,6 +57,15 @@
 	padding: 0;
 }
 
+// List Item Reset
+// This is mostly for Drupal projects to override the
+// default item links styles
+@mixin list-item-reset() {
```

Comment:
@johncionci can't you just use the mixin above? Why have two that do the same thing?